ros/src/computing/planning/decision/packages/decision_maker/CMakeLists.txt

```diff
@@ -85,3 +83,9 @@ install(DIRECTORY launch
         DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION}/launch/
         PATTERN ".svn" EXCLUDE
         )
+
+# Install config files
+foreach(dir config)
```

Comment:
there is no `config` directory to be installed.
https://travis-ci.org/CPFL/Autoware/jobs/498668551#L1278